src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime-project.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
We have over 100 projects with the closure burned in. We can't know which packages to drop/keep unless we look at the actual ref's. Tool will look at the refs and spit out minimum deps.  Bonus if I can get it to spit out min version of the deps, despite the refs being at a higher version.